services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.js

```diff
@@ -62,6 +62,19 @@ const CatalogMultiselectFilter = ({
     return sum + filter[item].length
   }, 0)
 
+  const addSpacerColumns = () => {
+    if (Object.keys(availableFilters).length % columns > 0) {
+      const spacerCols = []
```

Comment:
just curious if you think it's worth extracting `Object.keys(availableFilters).length` into a variable ?